Author: Pino Toscano <toscano.pino@tiscali.it>
Description: Enable fromBytesToWord on big endian architectures.
 fromBytesToWord is used in big endian code paths of KeccakF-1600-opt64.c
 (for 64bit architectures).
Last-Update: 2013-08-11
Forwarded: no

--- a/src/3rdparty/sha3/KeccakF-1600-opt64.c
+++ b/src/3rdparty/sha3/KeccakF-1600-opt64.c
@@ -324,7 +324,7 @@ static void KeccakPermutation(unsigned c
     KeccakPermutationOnWords((UINT64*)state);
 }
 
-#if 0 // Unused in the Qt configuration
+#if (PLATFORM_BYTE_ORDER == IS_BIG_ENDIAN)
 static void fromBytesToWord(UINT64 *word, const UINT8 *bytes)
 {
     unsigned int i;
